test/manual/amp-base-carousel/default-attributes.amp.html

```diff
@@ -0,0 +1,55 @@
+    <script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel-0.1.js"></script>
+    <style amp-boilerplate> body { -webkit-animation: -amp-start 8s steps(1, end) 0s 1 normal both; -moz-animation: -amp-start 8s steps(1, end) 0s 1 normal both; -ms-animation: -amp-start 8s steps(1, end) 0s 1 normal both; animation: -amp-start 8s steps(1, end) 0s 1 normal both; } @-webkit-keyframes -amp-start { from { visibility: hidden; } to { visibility: visible; } } @-moz-keyframes -amp-start { from { visibility: hidden; } to { visibility: visible; } } @-ms-keyframes -amp-start { from { visibility: hidden; } to { visibility: visible; } } @-o-keyframes -amp-start { from { visibility: hidden; } to { visibility: visible; } } @keyframes -amp-start { from { visibility: hidden; } to { visibility: visible; } } </style> 
+    <noscript> <style amp-boilerplate> body { -webkit-animation: none; -moz-animation: none; -ms-animation: none; animation: none; } </style> </noscript> 
+    <link rel="canonical" href="." />
+    <title>My AMP Page</title>
+    <style amp-custom>
+      body {
+        max-width: 527px;
+      }
+
+      amp-base-carousel img {
+        object-fit: cover;
+      }
+
+      .slide {
+        padding: 4px;
+      }
+    </style>
+  </head>
+
+  <body>
+    <!-- Default loop to false -->
+    <amp-base-carousel id="carousel-1" class="carousel" height="440" width="1000" layout="responsive">      
+      <amp-img  height="440" width="1000" class="slide" src="../img/redgradient.png" layout="responsive">Slide 1</amp-img>
+      <amp-img  height="440" width="1000" class="slide" src="../img/greengradient.png" layout="responsive">Slide 2</amp-img>
+      <amp-img  height="440" width="1000" class="slide" src="../img/bluegradient.png" layout="responsive">Slide 3</amp-img>
+      <amp-img  height="440" width="1000" class="slide" src="../img/orangegradient.png" layout="responsive">Slide 4</amp-img>
+    </amp-base-carousel>
+
+    <!-- Valid loop=true -->
+    <amp-base-carousel id="carousel-2" class="carousel" loop="true" visible-count="2" height="440" width="1000" layout="responsive">      
+      <amp-img  height="440" width="1000" class="slide" src="../img/redgradient.png" layout="responsive">Slide 1</amp-img>
+      <amp-img  height="440" width="1000" class="slide" src="../img/greengradient.png" layout="responsive">Slide 2</amp-img>
+      <amp-img  height="440" width="1000" class="slide" src="../img/bluegradient.png" layout="responsive">Slide 3</amp-img>
+      <amp-img  height="440" width="1000" class="slide" src="../img/orangegradient.png" layout="responsive">Slide 4</amp-img>
+      <amp-img  height="440" width="1000" class="slide" src="../img/tealgradient.png" layout="responsive">Slide 5</amp-img>
+      <amp-img  height="440" width="1000" class="slide" src="../img/lilacgradient.png" layout="responsive">Slide 6</amp-img>
+    </amp-base-carousel>
+
+    <!-- Invalid loop=true turned to false -->
```

Comment:
nit: could you describe inline why? it could be helpful for future people reading the code to be notified that `loop=true` is invalid because there are less than `3 * visible-count` slides provided